test/system/090-events.bats

```diff
@@ -79,11 +79,6 @@ function _events_disjunctive_filters() {
     _events_disjunctive_filters --events-backend=file
 }
 
-@test "events with disjunctive filters - journald" {
-    skip_if_remote "remote does not support --events-backend"
```

Comment:
Why not just `skip_if_journald_unavailable` ?

Reply:
Because I didn't know that existed :smile: 

Concur that's better. I'll also make this change against main.